lib/rubocop/cop/method_complexity_patch.rb

```diff
@@ -0,0 +1,10 @@
+# Monkey patching! Here be dragons.
+module RuboCop
+  module Cop
+    module MethodComplexity
+      def add_offense(node, loc, message = nil, severity = nil)
```

Comment:
What do you think of adding a comment here about what/why this is? Similar to what you have in the commit message